manifests/init.pp

```diff
@@ -239,6 +239,10 @@
 # @param service_enable
 #   Patroni service enable property
 # @param custom_pip_provider
+# @param  tags
```

Comment:
Please add an acceptance test that shows when this is set to true and has tags_parameters that the correct information is in the yaml config.